actions/pr-comment/package.json

```
@@ -0,0 +1,43 @@
+{
+  "name": "@pagopa/pr-comment-action",
+  "type": "module",
+  "version": "1.0.0",
+  "description": "TypeScript action for creating and managing PR comments",
+  "main": "dist/index.cjs",
+  "scripts": {
+    "build": "tsup",
+    "dev": "tsup --watch",
+    "typecheck": "tsc --noEmit",
+    "lint": "eslint --fix src",
+    "lint:check": "eslint src",
+    "format": "prettier --write src",
+    "format:check": "prettier --check src",
+    "test": "vitest run",
+    "test:watch": "vitest",
+    "test:coverage": "vitest run --coverage"
+  },
+  "keywords": [
+    "github",
+    "action",
+    "pr",
+    "comment",
+    "typescript"
+  ],
+  "author": "PagoPA DX Team",
+  "license": "MIT",
```

We haven't chosen a license yet

```suggestion
  "license": "UNLICENSED",
```